### ISSUE TITLE
Can't start a LAN game properly

### ISSUE THREAD

**Issue by Astolfo2001** [aldehir/ut2004-patches#18]
a few days ago, I tried to host a multiplayer game with somebody on Discord via 2 Virtual LAN programs: Radmin and ZeroTier

for the latter, my game wasn't showing in the server browser despite us both being connected to it

and for the former, every time it tried starting the game, it would either crash or disconnect us both instead

what is wrong and how do I fix it?

**Comment by aldehir** [comment]
@Astolfo2001,

UT2004, like many games in its era, use broadcasting to advertise itself on LAN networks. This may pose a problem when using a VPN solution, since it doesn't know which network interface to broadcast.

You're better off connecting directly via IP of the person hosting the game instance.

Regarding the crashes, do you have any logs of this occurring? You should have log files inside your UT2004 System folder.

**Comment by Astolfo2001** [comment]
I don't seem to have any logs of the crash.

**Comment by Astolfo2001** [comment]
my log files:
[ut2004-win64.log](https://github.com/user-attachments/files/16277239/ut2004-win64.log)
[UT2004.log](https://github.com/user-attachments/files/16277240/UT2004.log)
[Editor.log](https://github.com/user-attachments/files/16277241/Editor.log)
[server.log](https://github.com/user-attachments/files/16277242/server.log)

**Comment by Astolfo2001** [comment]
however 
I have heard of people getting Radmin and Hamachi to work with the game though:
https://radmin-club.com/other-games/unreal-tournament-2004-lan-play-setup/

**Comment by aldehir** [comment]
Yes, notice the requirement to set the interface metric so broadcast traffic is routed through the VPN interface instead of your default network interface. Nonetheless, LAN over a VPN is outside the scope of this project.

Your log only shows crashes for the win64 client, which these patches do not currently support. Try using the win32 client, which is `UT2004.exe` and not `ut2004-win64.exe` and post any crash logs encountered from that.

I also noticed that you're using the relics mutator. Because of the sheer amount of assets they contain you are forced to use the win64 client or patch the win32 client to enable large address aware (LAA). See https://www.techpowerup.com/forums/threads/large-address-aware.112556/ for more information if you want to patch the win32 client.
